includes/class-sensei-settings.php

```diff
@@ -162,6 +163,13 @@ public function init_sections() {
 			'description' => __( 'Settings for public Student Profiles.', 'sensei-lms' ),
 		);
 
+		if ( ! apply_filters( 'sensei_settings_woocommerce_hide', false ) ) {
+			$sections['woocommerce-settings'] = array(
+				'name'        => __( 'WooCommerce', 'sensei-lms' ),
+				'description' => __( 'Settings for public Student Profiles.', 'sensei-lms' ),
```

Comment:
The description isn't correct here. We should borrow [WCPC's description](https://github.com/Automattic/sensei-wc-paid-courses/blob/c1e49ee88054258be5dd87923ba61366799eedca/includes/class-settings.php#L55).

Reply:
[Fixed](https://github.com/Automattic/sensei/pull/4801/files#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR169)